From 11bef00e6dfbadcc772115ded7381b2de954692d Mon Sep 17 00:00:00 2001
From: Marian Chereji <marian.chereji@freescale.com>
Date: Tue, 14 May 2013 18:25:21 +0300
Subject: [PATCH 154/430] dpa_offload: Bug fix. Remove unnecessary fm_pcd
 attribute from compat parameters structure

There was one fm_pcd attribute that was left behind in the classifier
table parameters that seems to mess up the ioctl command codes. This
was removed.

Signed-off-by: Marian Chereji <marian.chereji@freescale.com>
Change-Id: I81cd6f32fc77f1bfa23ea72e814879f6bd00ab74
Reviewed-on: http://git.am.freescale.net:8181/2484
Reviewed-by: Bulie Radu-Andrei-B37577 <Radu.Bulie@freescale.com>
Reviewed-by: Fleming Andrew-AFLEMING <AFLEMING@freescale.com>
Tested-by: Fleming Andrew-AFLEMING <AFLEMING@freescale.com>
[original patch is from QorIQ-SDK-V1.4-20130625-yocto]
Signed-off-by: Jiang Lu <lu.jiang@windriver.com>
---
 .../staging/fsl_dpa_offload/dpa_classifier_ioctl.h |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

diff --git a/drivers/staging/fsl_dpa_offload/dpa_classifier_ioctl.h b/drivers/staging/fsl_dpa_offload/dpa_classifier_ioctl.h
index 4aab2d7..cc5f007 100644
--- a/drivers/staging/fsl_dpa_offload/dpa_classifier_ioctl.h
+++ b/drivers/staging/fsl_dpa_offload/dpa_classifier_ioctl.h
@@ -198,7 +198,6 @@ struct ioc_dpa_cls_mcast_remove_params {
 #ifdef CONFIG_COMPAT
 
 struct dpa_cls_compat_tbl_params {
-	compat_uptr_t			fm_pcd;
 	compat_uptr_t			cc_node;
 	compat_uptr_t			distribution;
 	compat_uptr_t                   classification;
-- 
1.7.5.4

